src/cola/wraptext.js

```diff
@@ -10,7 +10,7 @@ export default function wrapText(ctx, text, maxWidth) {
   do {
     let fits = false;
     const m = ctx.measureText(lines[activeLine]);
-    if (m.width > maxWidth) {
+    if (m.width > maxWidth && lines[activeLine].length > 1) {
```

Comment:
This fixes a problem I kept running into during game generation. In cases where the first or last word of a category string was longer than the [maxWidth](https://github.com/kesne/jeopardy-bot/blob/master/src/cola/generator.js#L136) constant passed into this function, the wrapper would fail because it would recursively try to wrap that same word over and over. e.g:

Category name: "TERMINATOR OF ENDEARMENT"
`canvas.measureText('ENDEARMENT')` resulted in something like 189 where `maxWidth` was 180.

This was happening quite often for me. I'm wondering if maybe the font sizes are rendered differently on your system. Anyway, this fix just ensures there is actually an extra word to wrap. The other option maybe is to just increase the value of maxWidth, but I think this is safer.

Tested this fix a number of times on games that were failing before. Game board still looks pretty good.